src/physics.js

```diff
@@ -1052,6 +1052,12 @@ AFRAME.registerComponent('physx-body', {
       }
     }
 
+    if (this.data.type === 'kinematic') {
+      if(this.data.highPrecision){
+        this.rigidBody.setRigidBodyFlag(PhysX.PxRigidBodyFlag.eENABLE_SPECULATIVE_CCD, true);
```

Comment:
I'd prefer to structure this code as 

```
if (high precision) {
  if (dynamic)...
  if (kinematic)...
}
```

For dynamic objects, we also do this:

```
this.rigidBody.setSolverIterationCounts(4, 2);
```

Is that needed / desirable for kinemtic objects as well?